### PR TITLE
Adds the Leviathan from Taxi (Asteroid) as a stand alone .dmm for away mission use

### PR DESCRIPTION
**Pull request by Probe1**
Adds a Gateway.
Adds new areas for it.
Tweaks Research Outpost to be suitable for an away mission
Moves abandoned mining outpost to be closer to the center
Deletes Vox Trade Outpost

Deletes all docking ports and such

To be honest I did a pretty lazy job with this but it works.  It's Saturday afternoon and I'm coding without pants on in multiple senses.

It takes an additional **FIFTY SECONDS** to load the game when this away mission is selected.

Screenshots:
https://github.com/d3athrow/vgstation13/pull/9797#issuecomment-215996898
https://github.com/d3athrow/vgstation13/pull/9797#issuecomment-215996925

### CODE REVIEW

**Comment by PJB3005** [comment]
What's the Leviathan?

Is it the squid asteroid?

**Comment by Probe1** [comment]
Yeah.  You're too fast for me.

**Comment by Probe1** [comment]
Abandoned mining outpost moved further south.

dreammaker_2016-04-30_17-31-45

**Comment by Probe1** [comment]
Research Outpost basically the same except for modifications to areas, apcs, cables, and the removal of xenoarch rp machines in favor of a gateway.

dreammaker_2016-04-30_17-31-51

File: code/game/area/Space Station 13 areas.dm [hunk 1]
```diff
@@ -2129,6 +2129,25 @@ proc/process_adminbus_teleport_locs()
 	lighting_use_dynamic = 0
 	requires_power = 0
 
+/area/awaymission/leviathan
+	name = "Leviathan"
+	icon_state = "null"
```
Comment (Intigracy) [inline code/game/area/Space Station 13 areas.dm]: just leave this as the inherited, null icon_state for areas isn't fun

Comment (Probe1) [inline code/game/area/Space Station 13 areas.dm]: it sounded fun to me.

**Comment by ghost** [comment]
Wouldn't it make sense to have the gateway where the miningresearch shuttle would dock, save there being unused space while bolting a new room on just for the gateway?

**Comment by Probe1** [comment]
Probably yes.

**Pull request merged**